docs/index.html

```diff
@@ -54,6 +54,7 @@ <h2 class="title">A 1x Engineer...</h2>
         <li>Isn't defined by myopic Tweetstorms by clueless VCs.</li>
         <li>Fails from time to time.</li>
         <li>Working to live, rather than living to work.</li>
+        <li>Can code even using a cheap laptop/PC</li>
```

Comment:
The way it's currently phrased could be read as an assertion of skill – "ha! I can code on ANY computer" – compared to the exact computer being an irrelevant instrument for code to be created. How about this? 

```suggest
        <li>Is not defined by the computer they're using.</li>
```

Reply:
thx,  good addition. I prefer ur option